src/utils.ts

```diff
@@ -250,6 +250,26 @@ export function getPlanFee(plan: Plan): number {
   return plan.type === 'prepaid' ? (plan as PrepaidPack).price : (plan as FeaturedPlan).fee;
 }
 
+const firstPlansDefault: PlanType[] = ['subscribers', 'monthly-deliveries', 'prepaid'];
```

Comment:
How about indicating in this variable that we are talking about order or sorting?
```suggestion
const firstPlansDefaultOrder: PlanType[] = ['subscribers', 'monthly-deliveries', 'prepaid'];
```

Reply:
great! applied!